query/src/sql/statements/statement_create_user_stage.rs

```diff
@@ -61,6 +62,7 @@ impl AnalyzableStatement for DfCreateUserStage {
                     &self.location,
                     &self.credential_options,
                     &self.encryption_options,
+                    &self.endpoint_url,
```

Comment:
Here is a consideration of accepting the user's endpoint.

Users could input endpoints like `127.0.0.1` to try to access our internal infra. We need to detect and avoid those things.